src/schism/schism_nuopc_cap.F90

```diff
@@ -325,6 +325,16 @@ subroutine InitializeAdvertise(comp, importState, exportState, clock, rc)
   write(message, '(A)') trim(compName)//' meshloc is set to '//trim(cvalue)
   call ESMF_LogWrite(trim(message), ESMF_LOGMSG_INFO)
 
+  ! debug option
+  call NUOPC_CompAttributeGet(comp, name='dbug', value=cvalue, isPresent=isPresent, isSet=isSet, rc=rc)
```

Comment:
I am not happy with 'dbug'.  That attribute should have a proper name, like 'debug',  or could even be a 'debug level' to allow more fine-grained debugging (later).  Also 'cvalue' is not good, should be logicalValue, e.g.

Reply:
@platipodium Okay. The issue is that to use RT ufs.configure template file for all the model components, I am using same attribute in the all components. If I change in here, I also need to change in other components to be consistent. Anyway, I'll do it soon and change it to debug level. The value is required since everything in ESMF config side is string.

Reply:
I changed to debug_level and the default is 0 - no debug output such as writing mesh, import and export states in VTK format